src/STEMMUS_SCOPE.m

```diff
@@ -815,7 +797,38 @@
             hSAVE = hN;
         end
     end
-    run Forcing_PARM;
+    %% Reconstruct the driving forces to be a continuous function of time
+    if TIMEOLD == KT
```

Comment:
~~So `KT` is the number of time steps, which is 0 in ModelSettings (??).~~
In `io.ModelSettings`, `KT` is set to 0, and is described as "the number of time steps". I think this is misleading, as it is not a setting. It is the counter for the _current_ time step.

Also, `TIMEOLD` is always 0:
`TIMEOLD = 0;` (line 546).

<hr>

Anyway, the code here sets all the values of these parameters to 0 for the first time step, but then right after it it sets them to a certain value? :confused: 

So for just one variable, as example:
```matlab
if KT == 0
    Ta(KT) = 0;
end

Ta(KT) = ForcingData.Ta_msr(KT)
```

What is the point of the `if KT == 0; varname(KT)  = 0; end` statement?

Reply:
> What is the point of the `if KT == 0; varname(KT) = 0; end` statement?

Hi, Bart, this is for the initialization of the model before the time step starts.

Reply:
all the if stetmanets in `Forcing_PARM` are unused. Also, the `if TIMEOLD == KT` does not happen.